tests/std/tests/P0355R7_calendars_and_time_zones_dates/test.cpp

```diff
@@ -967,6 +967,7 @@ constexpr void year_month_day_last_test() {
     if (is_constant_evaluated()) {
         static_assert((2020y / 1 / last).ok());
         static_assert(!(2020y / 13 / last).ok());
+        static_assert(!(2020y / 0 / last).day().ok());

```

Comment:
Should this have a comment that the `day()` is unspecified, and we're assuming that our implementation returns one that's `!ok()`? `// implementation-specific assumption, see GH-1647` would be sufficient.